cmdutils/flags.go

```diff
@@ -0,0 +1,33 @@
+package cmdutils
```

Comment:
can you put this package under `cmd`? Seems too low value to be a top level package

Reply:
I was thinking the same, but if I do `cmd/util`, people may think `util` is a command? What do you think?

Reply:
i think its ok, if they try to run it, it wont work since it doesnt have a main :D

Reply:
Hahaha okay